events/events-publisher/src/main/kotlin/uk/ac/ebi/events/service/EventsPublisherService.kt

```diff
@@ -25,18 +23,10 @@ class EventsPublisherService(
         val submissionNotification = SubmissionSubmitted(
             accNo = submission.accNo,
             uiUrl = instanceBaseUrl,
+            eventTime = Instant.now(),
             pagetabUrl = "$instanceBaseUrl/submissions/${submission.accNo}.json",
             extTabUrl = "$instanceBaseUrl/submissions/extended/${submission.accNo}",
             extUserUrl = "$instanceBaseUrl/security/users/extended/$ownerEmail")
-
         rabbitTemplate.convertAndSend(BIOSTUDIES_EXCHANGE, SUBMISSIONS_ROUTING_KEY, submissionNotification)
     }
-
-    fun requestSubmitted(submission: ExtSubmission, fileMode: FileMode) {
```

Comment:
Why are you removing this? Isn't this used in your previous changes?

Reply:
yes was not used